django_netjsongraph/visualizer/generics.py

```diff
@@ -19,7 +19,11 @@ def get(self, request, pk):
         topology = get_object_or_404(self.topology_model, pk)
         return render(request, 'netjsongraph/detail.html', {
             'topology': topology,
-            'graph_url': reverse('network_graph', args=[topology.pk]),
-            'history_url': reverse('network_graph_history', args=[topology.pk]),
+            'graph_url': reverse('network_graph',
+                                 urlconf=TOPOLOGY_API_URLCONF,
+                                 args=[topology.pk]),
+            'history_url': reverse('network_graph_history',
```

Comment:
what about the base URL? is this enough to make the application work also if the API has a different host?
Would this `reverse` call generate a path which would imply requests to the same host?